### PR TITLE
Command interface

### PR DESCRIPTION
**Pull request by whilefoo**
Related to https://github.com/ubiquity-os/ubiquity-os-kernel/pull/186

### CODE REVIEW

**Comment by gentlementlegen** [comment]
@whilefoo I think you forgot to update the bun.lockb file

**Comment by gentlementlegen** [comment]
@whilefoo you need to commit the bun lock file so jest tests can run 😄

File: package.json [hunk 1]
```diff
@@ -79,11 +87,13 @@
     "@octokit/rest": "^21.0.2",
     "@octokit/types": "^13.6.1",
     "@octokit/webhooks": "^13.3.0",
-    "@sinclair/typebox": "^0.33.21",
     "@ubiquity-os/ubiquity-os-logger": "^1.3.2",
     "dotenv": "^16.4.5",
     "hono": "^4.6.9"
   },
+  "peerDependencies": {
+    "@sinclair/typebox": "0.34.3"
```
Comment (gentlementlegen) [inline package.json:95]: peerDeps are great but yarn does not resolve them automatically afaik. Why did you move it here?

File: src/actions.ts [hunk 2]
```diff
@@ -18,29 +21,34 @@ interface Options {
   postCommentOnError?: boolean;
   settingsSchema?: TAnySchema;
   envSchema?: TAnySchema;
+  commandSchema?: TAnySchema;
   kernelPublicKey?: string;
+  disableSignatureVerification?: boolean; // only use for local development
```
Comment (gentlementlegen) [inline src/actions.ts:26]: maybe change the comment to
```
/*
**
*/
```
and add a deprecation or a warning so the IDE warns the user when settings this variable.

File: package.json (hunk 1 above)
Comment (whilefoo) [reply package.json:95]: As I understand it will warn the user if they are trying to install a typebox with version that doesn't match SDK's version

Comment (gentlementlegen) [reply package.json:95]: Correct but then it is the responsibility of the user to install TypeBox. However since we use it everywhere I believe it should be fine.

**Comment by gentlementlegen** [comment]
I am ok with the changes. Can you fix the CI and merge?

**Comment by whilefoo** [comment]
It turns out my Bun version was outdated so that's why it was complaining about the lockfile

**Review by gentlementlegen — APPROVED** [review]
